src/OrbitClientData/ProcessData.cpp

```diff
@@ -17,7 +17,43 @@ using orbit_grpc_protos::ModuleInfo;
 
 ProcessData::ProcessData() { process_info_.set_pid(-1); }
 
+void ProcessData::SetProcessInfo(const orbit_grpc_protos::ProcessInfo& process_info) {
+  absl::MutexLock lock(&mutex_);
+  process_info_ = process_info;
+}
+
+int32_t ProcessData::pid() const {
+  absl::MutexLock lock(&mutex_);
+  return process_info_.pid();
```

Comment:
Would it make sense to use `ReaderMutexLock`s and `WriterMutexLock`s as most accesses are reads?

Reply:
access to this class is not contentious enough to warrant suing reader/writer locks. Let's keep it as it is.